syntax = "proto2";
option optimize_for = SPEED;
option cc_generic_services = false;

import "google/protobuf/any.proto";

package NickSV.Chat;



message ProtoMessage
{
    optional uint32 sender_id = 1;
    optional string text = 2;
    optional google.protobuf.Any additional_data = 3;
}